Example/Example/ExampleViewController.swift

```diff
@@ -7,11 +7,45 @@
 //
 
 import UIKit
+import Ably
 
 class ExampleViewController: UIViewController {
 
+    private var client: ARTRealtime!
+
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        connectToAbly()
+    }
+
+    private func connectToAbly() {
+        /* Set up a Realtime client that authenticates with the local web server auth endpoint */
+        let options = ARTClientOptions()
+        options.authUrl = NSURL(string: "http://localhost:4567/auth")
```

Comment:
Is this IP address accessible from the mobile device?  It may be from an emulator although I doubt it, and definitely not from a mobile device where `localhost` is the device itself.

As this is not something that we can know in advance, I advise that we add a comment here about needing to change this IP to the public IP of the web server. Please test this out and provide suitable instructions here and in the tutorial itself.